examples/MultitenantBySubsections.xml

```diff
@@ -30,9 +30,10 @@
                   <FloorAreaValue>71017</FloorAreaValue>
                 </FloorArea>
               </FloorAreas>
-              <Subsections>
-                <Subsection ID="Suite180">
+              <Sections>
+                <Section ID="Suite180">
                   <PremisesName>Suite 180</PremisesName>
+                  <SectionType>Component</SectionType>
```

Comment:
Should we have a section type such as Tenant?

Reply:
I think so, that would be a logical value to add. I was thinking that "Single Tenant" would be better, but after some thought I'm not too sure.

Reply:
I will add this. I also think that the enums need to be sentenced cases... so `Single tenant`, `Whole building`, etc. thoughts?

Reply:
Yes, that's right, we have been using sentenced case.